sdk/storage/Azure.Storage.Common/tests/Shared/OpenWriteTestBase.cs

```diff
@@ -167,14 +183,14 @@ public async Task OpenWriteAsync_NewBlob()
         }
 
         [RecordedTest]
-        public async Task OpenWriteAsync_WithIntermediateFlushes()
+        public virtual async Task OpenWriteAsync_WithIntermediateFlushes()
         {
             // Arrange
             await using IDisposingContainer<TContainerClient> disposingContainer = await GetDisposingContainerAsync();
             TResourceClient client = GetResourceClient(disposingContainer.Container);
 
             // Act
-            using (Stream stream = await OpenWriteAsync(client, overwrite: true))
+            using (Stream stream = await OpenWriteAsync(client, maxDataSize: 175, overwrite: true))
```

Comment:
What is significant about 175?

Reply:
It's the sum of the numbers below. I can put all of those numbers in constants and just add them here